generic/tests/cfg/iozone_windows.cfg

```diff
@@ -4,11 +4,23 @@
     type = iozone_windows
     cdrom_cd1 = isos/windows/winutils.iso
     disk_letter = "C"
-    iozone_cmd = "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -n 1G -g 4G -M -b iozone.xls -f ${disk_letter}:\testfile"
+    iozone_cmd = "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -n 1G -g 4G -M -b iozone.xls -f %s:\testfile"
```

Comment:
Better not to change the original logic, since this case is called by others cases.
e.g.
```
./io-github-autotest-qemu/qemu/tests/cfg/driver_load_stress.cfg

    variants:
        - with_viostor:
            driver_name = viostor
            run_bgstress = iozone_windows
            drive_format_image1 = ide
            q35:
                drive_format_image1 = ahci
            cdrom_cd1 = isos/windows/winutils.iso
            images += " stg"
            image_name_stg = "images/storage"
            image_size_stg = 4G
            drive_format_stg = virtio
            force_create_image_stg = yes
            remove_image_stg = yes
            disk_letter = I
            disk_index = 1
            format_disk = yes
            iozone_cmd = "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -n 1G -g 2G -M -b iozone.xls -f ${disk_letter}:\testfile"
            target_process = iozone.exe
            iozone_timeout = 7200
```
if you wanna change it, make sure all the related cases are synced.

Reply:
that's ok~